Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.BeLowerCased.cs

```diff
@@ -68,11 +88,11 @@ public void When_a_non_lower_string_is_supposed_to_be_lower_it_should_fail_with_
 
             // Assert
             act.Should().Throw<XunitException>().WithMessage(
-                "Expected all characters in actual to be lower cased because we want to test the failure message, but found \"ABC\".");
+                "Expected all alpha characters in actual to be lower cased because we want to test the failure message, but found \"ABC\".");
```

Comment:
"alpha" -> "alphabetic" in four tests to make the failing tests pass

Reply:
Doh! Thought I'd done that. Thanks.
Where can I see that in the build results?